pkg/testutils/threadevents.go

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package testutils
+
+import (
+	"regexp"
+	"strconv"
+)
+
+type ThreadTesterInfo struct {
+	ParentPid, ParentTid                     uint32
+	Child1Pid, Child1Tid, ParentChild1Pid    uint32
+	Thread1Pid, Thread1Tid, ParentThread1Pid uint32
+}
+
+var (
+	ParentStartsRe = regexp.MustCompile(`parent:\t\t\(pid:(\d+), tid:(\d+), ppid:(\d+)\)\tstarts`)
+	Child1Re       = regexp.MustCompile(`Child 1:\t\(pid:(\d+), tid:(\d+), ppid:(\d+)\)\t`)
+	Thread1Re      = regexp.MustCompile(`Thread 1:\t\(pid:(\d+), tid:(\d+), ppid:(\d+)\)\t`)
+)
+
+func (tti *ThreadTesterInfo) ParseLine(l string) error {
```

Comment:
Is there a reason for adding this to testutils? Are there any other users than threads_test?

Reply:
Ah yes, in the kprobe test of the follow up PR  https://github.com/cilium/tetragon/pull/946/commits/4bee439cee92c5447055a1c1390e02013e8875b7  , guess that's fine